### ISSUE TITLE
Abalone metatdata: units

### ISSUE THREAD

**Issue by kinnla** [uci-ml-repo/ucimlrepo-feedback#59]
Thanks for providing the abalone data set! We are puzzled by the units given in the description of the abalone data set (abalone.names). For example the feature "Length": it is stated that the values that range from 0.075 to 0.815 are given in Milimeter. But the average size of an abalone, depending on species, varies from 20mm to 300mm, according to wikipedia. So from the domain perspective it seems likely that the values are given as foot, which would result in a range of 22.86 mm to 248.412 mm.

Could you please double check? We plan to use the data set for teaching materials, and don't want to spread misinformation about the beautiful sea snails.

**Comment by rlongjohn** [comment]
Thank you for your question! In the dataset information section, it reads, "the ranges of the continuous values have been scaled for use with an ANN (by dividing by 200)." So rather than these values actually being in feet, the conversion is to multiply the values in the dataset by 200 to get the values in millimeters. 

Also in the abalone.names file is a citation for the original dataset donor's PhD thesis (from 1995!); I found it available online [here](https://figshare.utas.edu.au/articles/thesis/Extending_and_benchmarking_Cascade-Correlation_extensions_to_the_Cascade-Correlation_architecture_and_benchmarking_of_feed-forward_supervised_artificial_neural_networks/23242847/1). I took a look through this to verify the pre-processing steps, and it looks like multiplying by 200 would indeed match up with the original ranges in mm reported in Table 6.3, i.e., 0.075 x 200 = 15 and 0.815 x 200 = 163.

Hope that helps!
